crates/cxx-qt-lib/src/core/qt.rs

```diff
@@ -22,6 +22,22 @@ mod ffi {
         CaseSensitive,
     }
 
+    #[repr(i32)]
+    enum ConnectionType {
```

Comment:
Please add a note here that UniqueConnection is **not** supported, because we're currently always using lambdas as functors.

Reply:
I think your comment may have been cut off there :thinking: 
See my suggestion.

Reply:
Done